prime-angular-frontend/src/app/modules/adjudication/adjudication-routing.module.ts

```diff
@@ -107,15 +107,16 @@ const routes: Routes = [
             data: { title: 'Site Registrations' }
           },
           {
+            // path: ':id',
```

Comment:
Note for reviewers.  Kept this so I can use the commented out code in the PEC ticket.